selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/social/SpaceManagement.java

```diff
@@ -220,6 +279,234 @@ public void saveChangesSpace() {
+            }
+        }
+        if (space != null) {
+            final SelenideElement ELEMENT_SPACE_NAME = $(byXpath("//div[@id='CalendarContainer']/div[@class='eventTitle']/span[@id and contains(text(),'$Space')]".replace("$Space", space)));
+            ELEMENT_SPACE_NAME.waitUntil(Condition.visible,Configuration.timeout);
+            info("Space Name is : " + space);
+            assertEquals(space, ELEMENT_SPACE_NAME.getText());
+        }
+        if (titleEvent != null) {
+            final SelenideElement ELEMENT_TITLE_EVENT_NAME = $(byXpath("//div[@class='pull-left eventSummary']/a[contains(text(),'$TitleEvent')]".replace("$TitleEvent", titleEvent)));
+            ELEMENT_TITLE_EVENT_NAME.waitUntil(Condition.visible,Configuration.timeout);
+            info("Title Event is : " + titleEvent);
+            assertEquals(titleEvent, ELEMENT_TITLE_EVENT_NAME.getText());
+        }
+        if (dateEvent != null) {
+            final String[] ELEMENT_CURRENT_DATE = $(byXpath("//div[@class='currentDateContainer']/center/a")).getText().split(": ");
+            info("Created Event Month is : ");
+            assertEquals(dateEvent.substring(6, 7), ELEMENT_CURRENT_DATE[1].substring(0, 1));
+            info("Created Event day is : ");
+            ArrayList<String> firstNineDays = new ArrayList<>();
+            firstNineDays.add("01");
+            firstNineDays.add("02");
+            firstNineDays.add("03");
+            firstNineDays.add("04");
+            firstNineDays.add("05");
+            firstNineDays.add("06");
+            firstNineDays.add("07");
+            firstNineDays.add("08");
+            firstNineDays.add("09");
+            if (firstNineDays.contains(dateEvent.substring(8,10))){
+                assertEquals(dateEvent.substring(9, 10), ELEMENT_CURRENT_DATE[1].substring(2, 3));
+                info("Created Event Year is : ");
+                assertEquals(dateEvent.substring(0, 4), ELEMENT_CURRENT_DATE[1].substring(4, 8));
+            }
+            else
+            {assertEquals(dateEvent.substring(8, 10), ELEMENT_CURRENT_DATE[1].substring(2, 4));
+            info("Created Event Year is : ");
+            assertEquals(dateEvent.substring(0, 4), ELEMENT_CURRENT_DATE[1].substring(5, 9));}
+
+        }
+        if (toolTips != null) {
+            info("Left arrow of the event calendar");
+            $(byXpath("//a[@class='actionIcon prevDate pull-left']")).waitUntil(Condition.visible, Configuration.timeout);
+            info("Right arrow of the event calendar");
+            $(byXpath("//i[@class='uiIconMiniArrowRight uiIconLightGray']")).waitUntil(Condition.visible, Configuration.timeout);
+        }
+    }
+
+    /**
+     * Check Manager Title Dimensions
+     */
+    public void checkManagerTitleDimensions() {
+        assertEquals("The line height is correct", "20px", ELEMENT_SPACE_MANAGER_NAME.getCssValue("line-height"));
```

Comment:
The first argument of the assertEquals method is the message used in logs in case of failure, so it should describe the error. For example here, it would be something like "The line height of space manager name is not 20px".

Reply:
Feedback not considered.

Reply:
Fixed done.